src/libslic3r/GCode.cpp

```diff
@@ -4197,9 +4325,7 @@ std::string GCode::extrude_loop(ExtrusionLoop loop, std::string description, dou
     Point last_pos = this->last_pos();
     if (!m_config.spiral_mode && description == "perimeter") {
         assert(m_layer != nullptr);
-        bool is_outer_wall_first = m_config.wall_infill_order == WallInfillOrder::OuterInnerInfill
-                                || m_config.wall_infill_order == WallInfillOrder::InfillOuterInner
-                                || m_config.wall_infill_order == WallInfillOrder::InnerOuterInnerInfill;
+        bool is_outer_wall_first = m_config.wall_sequence == WallSequence::OuterInner;
```

Comment:
Is this check sufficient to confirm Sandwich mode print order as is outer wall first? Or it doesnt matter?